cli/command/formatter/service.go

```diff
@@ -92,10 +92,10 @@ ContainerSpec:
 Mounts:
 {{- end }}
 {{- range $mount := .ContainerMounts }}
-  Target = {{ $mount.Target }}
-   Source = {{ $mount.Source }}
-   ReadOnly = {{ $mount.ReadOnly }}
-   Type = {{ $mount.Type }}
+ Target:	{{ $mount.Target }}
```

Comment:
I see the indentation changed; was this on purpose, or accidentally? (i.e. it should be indented under the `Mounts:` section)

Reply:
I changed the values identations to align values under each other. I guess that the keys are already matching the outer identation as other blocks (ie the keys identation under the Mounts: section)

Reply:
Ah, thanks, yes looks like Github showed it a bit odd